tutorials/resources.md

```diff
@@ -62,7 +62,7 @@ Gazebo will look for GUI plugins on the following paths, in order:
 2. [GUI plugins that are installed with Gazebo](https://github.com/gazebosim/gz-sim/tree/main/src/gui/plugins)
 3. Other paths added by calling `gz::gui::App()->AddPluginPath`
 4. `~/.gz/gui/plugins`
-5. [Plugins which are installed with Gazebo GUI](https://gazebosim.org/api/gui/4/namespace gz_1_1gui_1_1plugins.html)
+5. [Plugins which are installed with Gazebo GUI](https://gazebosim.org/api/gui/8/namespacegz_1_1gui_1_1plugins.html)
```

Comment:
This one doesn't work with `\ref` links.

Maybe the files defining plugins in `gz-gui/src/plugins/`need something like this in the source code to mark the namespace:
https://github.com/gazebosim/gz-sim/blob/9d79b629c9e0fed1c8635e30f06b2c61462f32d8/src/systems/ackermann_steering/AckermannSteering.hh#L29-L30
Probably won't change that now.

Reply:
Looking at the gz-gui8.tag.xml file, I expected to see
```xml
  <compound kind="namespace">
    <name>gz::gui::plugins</name>
    ...
```
but I don't. Adding 

```c++
    /// \brief Namespace for all plugins
    namespace plugins {
    }

```
to `gz-gui/include/gz/gui/Plugin.hh` does create the namespace in `gz-gui8.tag.xml`, so that's probably the necessary fix. I don't think the inline namespace is the issue.

Reply:
Okay. I'll put that single fix in another PR if that's okay, so that the fixes in this PR don't have to wait for the gz-gui fix.

Reply:
Welp, I just opened a PR https://github.com/gazebosim/gz-gui/pull/571

Reply:
`gz::gui::plugins` now works in 8c97df5ae
But this PR now depends on gz-gui#571 above. So don't merge this before that. Or I can break out this one change into a new PR if the gz-gui one takes too long.